docs/auth/auth0/provider.md

```diff
@@ -47,6 +47,7 @@ The Auth0 provider is a structure with three configuration keys:
 - `clientSecret`: The Application client secret, found on the Auth0 Application
   page
 - `domain`: The Application domain, found on the Auth0 Application page
+- `sessionsecret`: The session secret is a key used for signing and/or encrypting cookies set by the application to maintain session state. In this case, 'your session secret' should be replaced with a long, complex, and unique string that only your application knows.
```

Comment:
```suggestion
- `session.secret`: The session secret is a key used for signing and/or encrypting cookies set by the application to maintain session state. In this case, 'your session secret' should be replaced with a long, complex, and unique string that only your application knows.
```

It kinda feels like this should be documented at a higher level somewhere? This piece of configuration is independent of any particular auth provider...